src/tir/transforms/remove_no_op.cc

```diff
@@ -162,6 +168,9 @@ class NoOpRemover : public StmtMutator {
     }
     return stmt.defined() ? stmt : Evaluate(0);
   }
+
+  std::unordered_map<const VarNode*, arith::IntSet> var_range_map;
+  arith::Analyzer analyzer;
```

Comment:
```suggestion
  std::unordered_map<const VarNode*, arith::IntSet> var_range_map_;
  arith::Analyzer analyzer_;
```
Also, please add the document

Reply:
OK